projects/ion/src/lib/card/card.component.spec.ts

```diff
@@ -121,4 +132,21 @@ describe('CardComponent', () => {
     fixture.detectChanges();
     expect(screen.getByTestId('footerTest')).toBeInTheDocument();
   });
+
+  it('should not render chips by default', async () => {
+    cardComponent.cardConfig.header.chips = undefined;
+    fixture.detectChanges();
+    const chipsContainer = screen.queryByTestId('chips-container');
+    expect(chipsContainer).toBeNull();
+  });
+
+  it('should render chips', async () => {
```

Comment:
Check chips has displaying in screen by label, I suggest use of it.each(yourChipsArray) and expect element with chip label on screen.